packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -988,6 +988,23 @@ const runGenerator = async (
                 mobify: {app: {extensions: [answers.project.name]}}
             })
 
+            // TODO: The generator is growing, we should refactor this to be more maintainable.
```

Comment:
Is there better ways to handle this without introducing huge refactors in the generator?

Reply:
At least it's a little cleaner now with the meta file